library/std/src/keyword_docs.rs

```diff
@@ -707,8 +707,8 @@ mod impl_keyword {}
 ///
 /// ## Literal Examples:
 ///
-///    * `for _ **in** 1..3 {}` - Iterate over an exclusive range up to but excluding 3.
-///    * `for _ **in** 1..=3 {}` - Iterate over an inclusive range up to and including 3.
+///    * <code>for _ **in** 1..3 {}</code> - Iterate over an exclusive range up to but excluding 3.
+///    * <code>for _ **in** 1..=3 {}</code> - Iterate over an inclusive range up to and including 3.
```

Comment:
Should we just remove the bold? Even though it may remove the emphasis but looks pretty clear to me already, or maybe we bold the `in` keyword in the details part above?
```suggestion
///    * `for _ in 1..3 {}` - Iterate over an exclusive range up to but excluding 3.
///    * `for _ in 1..=3 {}` - Iterate over an inclusive range up to and including 3.
```

Reply:
I also thought of just removing the bold instead of writing HTML here. I'll do that.
I'm not sure where exactly you suggest to bold the `in` keyword in addition to this. I'm not very familiar with the docs.